### ISSUE TITLE
Enhancement: Sanity check during manual URL configuration

### ISSUE THREAD

**Issue by atom-box** [matomo-org/tag-manager#730]
Have a check in the tag manager to see if the Matomo URL stored in the database matches the one they have in the Tag Manager configuration.

Show an info or warning panel if it doesn't match.

**Comment by snake14** [comment]
Hi @atom-box . Thank you for taking the time to create this issue. For clarification, you're talking about having some form of warnings if the URL field in a Matomo configuration variable doesn't match the URL configured for the site?
![image](https://github.com/matomo-org/tag-manager/assets/11394099/fdb1bbc8-eab9-4a13-9674-353d463a6517)

**Comment by atom-box** [comment]
Yes, correct:  check if the tracking URL matches the Matomo URL.
